src/Flex.php

```diff
@@ -182,6 +182,27 @@ public function activate(Composer $composer, IOInterface $io)
         }
     }
 
+    private function getLockFilePath()
+    {
+        $composerFile = Factory::getComposerFile();
+        $lockFilePath = "json" === pathinfo($composerFile, PATHINFO_EXTENSION)
+                ? substr($composerFile, 0, -4).'lock'
+                :  $composerFile .  '.lock';
+        $symfonyLock = str_replace("composer.", "symfony.", $lockFilePath);
+        if (getenv("SYMFONY_LOCKFILE_PATH")) {
+            $symfonyLock  = getenv("SYMFONY_LOCKFILE_PATH");
+        }
+        return (object)[
+            "composer" => $lockFilePath,
+            "symfony" => $symfonyLock
```

Comment:
instead of returning an object with 2 properties where each caller only need 1, I would create 2 methods returning strings instead.

Reply:
valid point, thx! done!